README.md

```diff
@@ -330,6 +330,17 @@ Then you can run this in the checkout root to reformat all Rust code:
 $ make rustfmt
 ```
 
+### Running tests
+
+* make check
+  Run all tests as defined in the directory.  Expensive tests are
+  suppressed.  The result of the tests for <filename>.el is stored in
+  <filename>.log.
+
+* make check-maybe
+  Like "make check", but run only the tests for files which have
+  unresolved prerequisites.
```

Comment:
Perhaps worth saying what we mean by unresolved prerequisites. I presume this means files that have been modified since the last build (i.e. usual Makefile logic)?

Reply:
I actually don't know what it means, I just took it from the README in the test directory ;)

Reply:
@Wilfred you are reading that correctly. Make check looks for <file>.log which depends on <file>.elc which in turn depends on <file>.el.